asv_bench/benchmarks/categoricals.py

```diff
@@ -43,3 +45,16 @@ def time_regular_constructor(self):
     def time_fastpath(self):
         Categorical(self.codes, self.cat_idx, fastpath=True)
 
+
+class categorical_rendering(object):
+    goal_time = 2e-3
+
+    def setup(self):
+        n = 1000
+        items = [str(i) for i in range(n)]
+        s = pd.Series(items, dtype='category')
+        df = pd.DataFrame({'C': s, 'data': np.random.randn(n)})
+        self.data = df[df.C == '20']
+
+    def time_rendering(self):
+        str(data.C)
```

Comment:
Yeah, sorry. I can't run asv on my machine. first it complained about a missing config file, when i tried to run only the categorical bench test. apparently, when i'm not restricting it with a `-b` flag, it creates a new environment, but it seems to be incompatible with conda, as it creates a py2.7 environment now, while being run in a conda python3.4 environment.